packages/datetime/src/_daterangepicker.scss

```diff
@@ -55,11 +55,26 @@ Styleguide components.datetime.daterangepicker.js
 .pt-daterangepicker {
   white-space: nowrap;
 
+  &.pt-daterangepicker-sequential {
+    .DayPicker {
+      min-width: $pt-grid-size * 43;
```

Comment:
Extracted out the width for DayPicker since it's used to calculate the other widths. And made the css a bit more verbose to show what's going on. I haven't moved the shortcuts out, since it is arbitrary.